src/Symfony/Component/Validator/Constraints/Timezone.php

```diff
@@ -0,0 +1,26 @@
+<?php
+
+/*
+* This file is part of the Symfony package.
+*
+* (c) Fabien Potencier <fabien@symfony.com>
+*
+* For the full copyright and license information, please view the LICENSE
+* file that was distributed with this source code.
+*/
+
+namespace Symfony\Component\Validator\Constraints;
+
+use Symfony\Component\Validator\Constraint;
+
+/**
+ * @Annotation
+ * @Target({"PROPERTY", "METHOD", "ANNOTATION"})
+ *
+ * @author Mickaël Andrieu <andrieu.travail@gmail.com>
+ *
+ */
+class Timezone extends Constraint
+{
+    public $message = 'The value "{{ value }}" is not a valid timezone';
```

Comment:
This message should be also added to [the translations file](https://github.com/symfony/symfony/blob/ad2f906daf8543a7034ef2f0eebd4531084f5fbe/src/Symfony/Component/Validator/Resources/translations/validators.en.xlf#L309) (at least the English one).